### PR TITLE
feat: PeakEWMA supports configuring activeRequestBias and optimizes the default EWMA of newly added hosts

### PR DESCRIPTION
**Pull request by jizhuozhi**
### Issues associated with this PR

Since #2285 and #2286 makes `activeRequest` having a configurable bias, I want to make it effective in PeakEWMA

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/upstream/cluster/loadbalancer.go [hunk 1]
```diff
@@ -779,10 +788,22 @@ func (lb *peakEwmaLoadBalancer) unweightedPeakEwmaScore(h types.Host) float64 {
...
+		duration = h.ClusterInfo().Stats().UpstreamRequestDurationEWMA.Rate()
+		// None of the active requests returned in cluster, or the metrics is disabled
+		if duration == 0 {
+			duration = float64(lb.defaultDuration)
+		}
+	}
+
+	biasedActiveRequest := float64(stats.UpstreamRequestActive.Count()) + 1
+	if biasedActiveRequest == 1.0 || lb.activeRequestBias == 0.0 {
+		biasedActiveRequest = 1
```
Comment (spacewander) [inline pkg/upstream/cluster/loadbalancer.go:803]: Why set biasedActiveRequest to 1 when `if biasedActiveRequest == 1.0`?

Comment (jizhuozhi) [reply pkg/upstream/cluster/loadbalancer.go:803]: avoid too many branches

Comment (spacewander) [reply pkg/upstream/cluster/loadbalancer.go:803]: I take a look at Go's Pow implementation:
https://github.com/golang/go/blob/e335a2665f5e322a7da8baa22fe816b6ef9aaf24/src/math/pow.go#L57-L60
There are already additional special handling for these cases.

Do the additional branches in the caller's code improve the performance?

Comment (jizhuozhi) [reply pkg/upstream/cluster/loadbalancer.go]: Thank you very much, I haven't paid attention to the implementation of Pow before, here is a redundant optimization

**Review by spacewander — APPROVED** [review]

**Review by doujiang24 — APPROVED** [review]

**Pull request merged**